morse.rb

```diff
@@ -0,0 +1,39 @@
+print 'Type some morse code here: '
+morse_str = gets.chomp
```

Comment:
- [ ] I would recommend removing these lines and also all the puts line below from 34. As you don't need to ask to enter any, instead we will check directly.